ui/components/app/currency-input/currency-input.js

```diff
@@ -160,6 +165,9 @@ export default function CurrencyInput({
 
     return (
       <CurrencyDisplay
+        // hides the fiat suffix
+        hideLabel={isPrimary || isLongSymbol}
+        suffix={suffix}
```

Comment:
Is there ever a scenario in which CurrencyDisplay shows a fiat value with a suffix?